### DISCUSSION TITLE
Les funciona en el Theme de Neve? A mi no.

### DISCUSSION THREAD

**Discussion by BeShanti** [whooohq/whq-woocommerce-chilexpress-shipping#233] [Support]
Hola! Gracias por el trabajo invertido en este Plugin. El oficial no fue bien.
Este por ahora no me funciona pero le tengo fe.

Tengo todas las opciones activadas existentes de envío (retiro en persona / envío gratis / envío con costo fijo ). Permite cambiar la dirección de envío, pero no aparece la opción de Chilexpress como está en la página de ejemplo. 

Uso el theme Neve. Será por eso? A alguien más le pasó? 
Porque si no, no sabría que podría ser...

La página es beshanti.cl

Gracias desde ya, si me pudiesen dar alguna orientación. 

Un saludo.

**Comment by EstebanForge** [comment]
Buenas.

Sigue estos pasos https://github.com/whooohq/whq-woocommerce-chilexpress-shipping#problemas-generales-visuales-glitches-yo-funcionales-en-el-carro-o-finalizar-compra-checkout y reportas de vuelta como te fue, por favor.

Esteban

**Reply by BeShanti** [reply]
Gracias por la pronta contestación. 
Ya desactivé todos los otros plugins y cambié al Twenty Twenty-One.
Los plugins y Wordpress están actualizados.

Sigue sin aparecer la opción de Envío de Chilexpress en el Carrito. 
(Por un rato dejo el Twenty Twenty-One aunque no esté personalizado)

beshanti.cl

Gracias.

**Reply by EstebanForge** [reply]
Estimado, en ninguna parte de los pasos se menciona usar el Theme Twenty Twenty-One.

Por favor, revisa y lee bien lo solicitado, y sigue el paso a paso al pie de la letra. Gracias!

**Reply by BeShanti** [reply]
Si. Está con el Storefront (Ahora activo). Y el mismo problema.

**Reply by EstebanForge** [reply]
Revisando vuestro checkout, veo que ni siquiera tienes activo el plugin de Chilexpress que ofrecemos acá:
https://i.imgur.com/wN5DZ8r.png

Ninguna referencia al plugin de Chilexpress en el HTML de salida del checkout: https://i.imgur.com/EAqYQLW.png

**Reply by BeShanti** [reply]
Ok. Gracias. 
Se ve que hay algo que no estoy sabiendo hacer. Esto no es suficiente?
![Sin título](https://user-images.githubusercontent.com/97473352/148833088-327a601f-58d9-4ba4-b427-f6d2fa3dbba0.png)

**Reply by EstebanForge** [reply]
¿De casualidad esa es una instancia de WordPress Multisite Network? https://wordpress.org/support/article/create-a-network/

**Reply by BeShanti** [reply]
Según https://www.quora.com/How-can-I-find-out-if-a-wordpress-site-is-installed-as-a-multisite-or-not  no lo es.

**Reply by BeShanti** [reply]
Hola. Siguo sin poder hacerlo funcionar. Sigue con el Theme StoreFront y todo actualizado. El Plugin activado. Y reconfirmado que no es un multisite.

Este producto tiene todas las especificaciones de medidas y peso. (Por las dudas por si era eso, pero tampoco)
https://beshanti.cl/producto/muneca/

**Reply by BeShanti** [reply]
Es problema del plugin entonces? No funciona?
